core/genesis_test.go

```diff
@@ -182,15 +182,17 @@ func TestStatefulPrecompilesConfigure(t *testing.T) {
 		"allow list enabled in genesis": {
 			getConfig: func() *params.ChainConfig {
 				config := *params.TestChainConfig
-				config.AllowListConfig = precompile.AllowListConfig{
-					BlockTimestamp:  big.NewInt(0),
-					AllowListAdmins: []common.Address{addr},
+				config.ContractDeployerAllowListConfig = precompile.ContractDeployerAllowListConfig{
+					AllowListConfig: precompile.AllowListConfig{
+						BlockTimestamp:  big.NewInt(0),
```

Comment:
I think we should hoist `BlockTimestamp` into the top level?

Reply:
Not sure about this. This will be the same for any config that uses this.